vendor/github.com/fearful-symmetry/gomsr/go.mod

```diff
@@ -1 +1,3 @@
 module github.com/fearful-symmetry/gomsr
+
+go 1.13
```

Comment:
seems unrelated

Reply:
deleted. maybe a relic of other branch, i cant tell.